[ja] docs(i18n): translate content/en/docs/languages/js/*.md files

Msksgm opened the pull request:
<!--
Please provide a meaningful description of what your changes will do. Bonus points for including links to related issues, other PRs, or technical references.

A detailed description helps reviewers understand the context of your change and reduces
the time needed to review.

Please make sure to follow the [contribution guidelines](https://opentelemetry.io/docs/contributing/).

We run checks on all PRs, to ensure that your contribution follows our [style guide](https://opentelemetry.io/docs/contributing/style-guide/).

Many requirements of that guide can be enforced by running `npm run fix:all` locally and committing the changes.

Note, that although those checks need to be green before we can merge your PR, do not
worry about their status after submitting your PR. We will provide you with guidance, when and how to
address them.

Similarly, do not worry about your PR being out-of-date with the base branch and do not continuously
update or rebase your branch. We will let you know when it is necessary.

If you have any questions, feel free to ask.
-->

Original English pages.

- https://deploy-preview-7164--opentelemetry.netlify.app/docs/languages/js/getting-started/
- https://deploy-preview-7164--opentelemetry.netlify.app/docs/languages/js/getting-started/nodejs/
- https://deploy-preview-7164--opentelemetry.netlify.app/docs/languages/js/getting-started/browser/
- https://deploy-preview-7164--opentelemetry.netlify.app/docs/languages/js/instrumentation/
- https://deploy-preview-7164--opentelemetry.netlify.app/docs/languages/js/libraries/
- https://deploy-preview-7164--opentelemetry.netlify.app/docs/languages/js/exporters/
- https://deploy-preview-7164--opentelemetry.netlify.app/docs/languages/js/context/
- https://deploy-preview-7164--opentelemetry.netlify.app/docs/languages/js/propagation/
- https://deploy-preview-7164--opentelemetry.netlify.app/docs/languages/js/resources/
- https://deploy-preview-7164--opentelemetry.netlify.app/docs/languages/js/sampling/
- https://deploy-preview-7164--opentelemetry.netlify.app/docs/languages/js/serverless/
- https://deploy-preview-7164--opentelemetry.netlify.app/docs/languages/js/benchmarks/


Preview pages.

- https://deploy-preview-7164--opentelemetry.netlify.app/ja/docs/languages/js/getting-started/
- https://deploy-preview-7164--opentelemetry.netlify.app/ja/docs/languages/js/getting-started/nodejs/
- https://deploy-preview-7164--opentelemetry.netlify.app/ja/docs/languages/js/getting-started/browser/
- https://deploy-preview-7164--opentelemetry.netlify.app/ja/docs/languages/js/instrumentation/
- https://deploy-preview-7164--opentelemetry.netlify.app/ja/docs/languages/js/libraries/
- https://deploy-preview-7164--opentelemetry.netlify.app/ja/docs/languages/js/exporters/
- https://deploy-preview-7164--opentelemetry.netlify.app/ja/docs/languages/js/context/
- https://deploy-preview-7164--opentelemetry.netlify.app/ja/docs/languages/js/propagation/
- https://deploy-preview-7164--opentelemetry.netlify.app/ja/docs/languages/js/resources/
- https://deploy-preview-7164--opentelemetry.netlify.app/ja/docs/languages/js/sampling/
- https://deploy-preview-7164--opentelemetry.netlify.app/ja/docs/languages/js/serverless/
- https://deploy-preview-7164--opentelemetry.netlify.app/ja/docs/languages/js/benchmarks/

Msksgm commented:
@chalin 

Hello. I have a question. Bellow preview pages include English sentences because they includes like `{{% docs/languages/xxx %}}`. I think that  it is a similar issue of https://github.com/open-telemetry/opentelemetry.io/issues/6460.
Although I tried to address it by translating `layouts/_shortcodes/docs/languages/*.md` pages and set it in a new directory, I could not do it because I cannot identify the place where I had to set translated files. Could you give me any instructions?

- https://deploy-preview-7164--opentelemetry.netlify.app/ja/docs/languages/js/libraries/
- https://deploy-preview-7164--opentelemetry.netlify.app/ja/docs/languages/js/propagation/

ymotongpoo commented on content/ja/docs/languages/js/exporters.md:
```diff
@@ -0,0 +1,346 @@
+
+## Console {#console}
+
+計装をデバッグしたり、開発環境でローカルに値を確認したりするために、テレメトリーデータをコンソール（stdout）に書き込むエクスポーターを使用できます。
```
```suggestion
計装をデバッグしたり、開発環境でローカルに値を確認したりするために、テレメトリーデータをコンソール（標準出力）に書き込むエクスポーターを使用できます。
```

Msksgm replied:
ありがとうございます。以下で対応しました。

https://github.com/open-telemetry/opentelemetry.io/pull/7164/commits/259aa38ca462565d29c4668a894bd3b5de5e52b1

ymotongpoo commented on content/ja/docs/languages/js/exporters.md:
```diff
@@ -0,0 +1,346 @@
+
+計装をデバッグしたり、開発環境でローカルに値を確認したりするために、テレメトリーデータをコンソール（stdout）に書き込むエクスポーターを使用できます。
+
+[Getting Started](/docs/languages/js/getting-started/nodejs/)または[Manual Instrumentation](/docs/languages/js/instrumentation)ガイドに従った場合、コンソールエクスポーターはすでにインストールされています。
```
該当する見出しの日本語に変更

Msksgm replied:
ありがとうございます。以下で対応しました。

https://github.com/open-telemetry/opentelemetry.io/pull/7164/commits/d2632c66d594427c3184a4ade1958daf593baea1

ymotongpoo commented on content/ja/docs/languages/js/propagation.md:
```diff
@@ -0,0 +1,523 @@
+---
+title: プロパゲーション
```
伝搬で統一しています

Msksgm replied:
ありがとうございます。以下で対応しました。
textlint でも対応しますね。

https://github.com/open-telemetry/opentelemetry.io/pull/7164/commits/d51762e1235c53d71090fe8c23658a65d2a0c4cf

ymotongpoo commented on content/ja/docs/languages/js/propagation.md:
```diff
@@ -0,0 +1,523 @@
+
+## 自動コンテキストプロパゲーション {#automatic-context-propagation}
+
+[`@opentelemetry/instrumentation-http`](https://www.npmjs.com/package/@opentelemetry/instrumentation-http)や[`@opentelemetry/instrumentation-express`](https://www.npmjs.com/package/@opentelemetry/instrumentation-express)などの[インストルメンテーションライブラリ](../libraries/)は、サービス間でのコンテキストのプロパゲーションを自動的に行います。
```
計装

Msksgm replied:
ありがとうございます。言い訳なんですが、`[]` や フロントマターに含まれているものは textlint で検知されないですね。。。

https://github.com/open-telemetry/opentelemetry.io/pull/7164/commits/c44adb1cec3f4bcaa49481ff2679dd22855b839c

Msksgm commented on content/ja/docs/languages/js/context.md:
````diff
@@ -0,0 +1,228 @@
+console.log(ctx.getValue(key)); // undefined
+```
+
+## Active Context {#active-context}
````
ありがとうございます。
Context まわりの修正と合わせて対応しました。

https://github.com/open-telemetry/opentelemetry.io/pull/7164/commits/e9c375055eced70b34842e90f3134b0ffe2fe6fc

Msksgm commented on content/ja/docs/languages/js/exporters.md:
````diff
@@ -0,0 +1,346 @@
+}
+```
+
+## Console {#console}
````
ありがとうございます。以下で対応しました。

https://github.com/open-telemetry/opentelemetry.io/pull/7164/commits/7bf41afc00e7ca9d9265c99a79c980458a914e39

chalin commented:
> Hello. I have a question. Bellow preview pages include English sentences because they includes like `{{% docs/languages/xxx %}}`. I think that it is a similar issue of #6460.

Hi, I was away. Thanks for your patience in getting a response.

My suggestion is to commit this PR as is. We'll address those English parts in a followup PR. I'll convert that shortcode into an include and you will then be able to add a translated include file. How does that sound?

chalin commented:
@Msksgm, see:

- #7452
- #7453 

Once these are both merged, you'll be able to rebase this PR, and add the Japanese translations for the two new include files.

Msksgm commented:
@chalin 

> @Msksgm, see:
> 
> * [[i18n] Move alert prose from `native-libraries` partial into an include file #7452](https://github.com/open-telemetry/opentelemetry.io/pull/7452)
> * [[i18n] Move prose out of libraries-intro shortcode into include file #7453](https://github.com/open-telemetry/opentelemetry.io/pull/7453)
> 
> Once these are both merged, you'll be able to rebase this PR, and add the Japanese translations for the two new include files.

Thank you for your answer. I appreciate that you fix this issue quickly.
I rebased these PR and translated two files in the bellow commits.

- https://github.com/open-telemetry/opentelemetry.io/pull/7164/commits/33d81ab570ee5ee3044bca30c09a8b5953156906
- https://github.com/open-telemetry/opentelemetry.io/pull/7164/commits/0a712c57bd4416aed2a12658d48b09692e017ea2

I confirmed that https://deploy-preview-7164--opentelemetry.netlify.app/ja/docs/languages/js/libraries/ page is fully translated.

However, I think that English in https://deploy-preview-7164--opentelemetry.netlify.app/ja/docs/languages/js/propagation/ page is still remained. To resolve this issue, I think that https://github.com/open-telemetry/opentelemetry.io/blob/main/layouts/_shortcodes/docs/languages/propagation.md should be also moved to content/en/docs/languages/_includes/ directory.
Do I have to wait for this addressing?

vitorvasc commented:
> However, I think that English in https://deploy-preview-7164--opentelemetry.netlify.app/ja/docs/languages/js/propagation/ page is still remained. To resolve this issue, I think that https://github.com/open-telemetry/opentelemetry.io/blob/main/layouts/_shortcodes/docs/languages/propagation.md should be also moved to content/en/docs/languages/_includes/ directory. Do I have to wait for this addressing?

Hi @Msksgm! I think I see what’s going on here. @chalin is OOO for the next few weeks, but I can take over from here so we can get you unblocked on this PR 🙌🏼

Let me take a closer look, and hopefully I’ll be back soon with more details.

vitorvasc commented:
Hi @Msksgm! 

We've already merged #7511, which contains the changes needed for the `propagation.md` shortcode to be used as an include. Just like the previous changes, you should now be able to rebase this PR and add the Japanese translation for this file under the `_includes` directory.

Msksgm commented:
> Hi @Msksgm!
> 
> We've already merged #7511, which contains the changes needed for the `propagation.md` shortcode to be used as an include. Just like the previous changes, you should now be able to rebase this PR and add the Japanese translation for this file under the `_includes` directory.

@vitorvasc 

Thank you for your sharing!
I translated the page you mentioned and I've confirmed that https://deploy-preview-7164--opentelemetry.netlify.app/ja/docs/languages/js/propagation/ is fully translated.

Msksgm commented:
@ymotongpoo 

こちら、すべてのページで翻訳後が表示されるようになったので、確認のほどお願いします🙇

svrnm commented:
@open-telemetry/docs-ja-approvers PTAL!

ymotongpoo commented on content/ja/docs/languages/_includes/propagation.md:
```diff
@@ -0,0 +1,21 @@
+default_lang_commit: f9a0439ac56dba1515283e1a1cb6d6a90634a20f
+---
+
+コンテキスト伝播により、[シグナル](/docs/concepts/signals/)は、生成される場所に関係なく、相互に関連付けることができます。
```
```suggestion
コンテキスト伝搬により、[シグナル](/docs/concepts/signals/)は、生成される場所に関係なく、相互に関連付けることができます。
```

ymotongpoo commented on content/ja/docs/languages/_includes/propagation.md:
```diff
@@ -0,0 +1,21 @@
+---
+
+コンテキスト伝播により、[シグナル](/docs/concepts/signals/)は、生成される場所に関係なく、相互に関連付けることができます。
+トレーシングに限定されませんが、コンテキスト伝播により、[トレース](/docs/concepts/signals/traces/)は、プロセスとネットワークの境界を越えて任意に分散されたサービス間で、システムに関する因果関係の情報を構築できます。
```
```suggestion
トレーシングに限定されませんが、コンテキスト伝搬により、[トレース](/docs/concepts/signals/traces/)は、プロセスとネットワークの境界を越えて任意に分散されたサービス間で、システムに関する因果関係の情報を構築できます。
```

ymotongpoo commented on content/ja/docs/languages/_includes/propagation.md:
```diff
@@ -0,0 +1,21 @@
+コンテキスト伝播により、[シグナル](/docs/concepts/signals/)は、生成される場所に関係なく、相互に関連付けることができます。
+トレーシングに限定されませんが、コンテキスト伝播により、[トレース](/docs/concepts/signals/traces/)は、プロセスとネットワークの境界を越えて任意に分散されたサービス間で、システムに関する因果関係の情報を構築できます。
+
+大多数のユースケースでは、OpenTelemetryをネイティブにサポートするライブラリまたは[計装ライブラリ](../libraries/)が、自動的にサービス間でトレースコンテキストを伝播します。
```
```suggestion
大多数のユースケースでは、OpenTelemetryをネイティブにサポートするライブラリまたは[計装ライブラリ](../libraries/)が、自動的にサービス間でトレースコンテキストを伝搬します。
```

ymotongpoo commented on content/ja/docs/languages/_includes/propagation.md:
```diff
@@ -0,0 +1,21 @@
+トレーシングに限定されませんが、コンテキスト伝播により、[トレース](/docs/concepts/signals/traces/)は、プロセスとネットワークの境界を越えて任意に分散されたサービス間で、システムに関する因果関係の情報を構築できます。
+
+大多数のユースケースでは、OpenTelemetryをネイティブにサポートするライブラリまたは[計装ライブラリ](../libraries/)が、自動的にサービス間でトレースコンテキストを伝播します。
+手動でコンテキストを伝播する必要があるのは、まれなケースのみです。
```
```suggestion
手動でコンテキストを伝搬する必要があるのは、まれなケースのみです。
```

ymotongpoo commented on content/ja/docs/languages/_includes/propagation.md:
```diff
@@ -0,0 +1,21 @@
+
+{{ if $lang }}
+
+詳細については、[コンテキスト伝播](/docs/concepts/context-propagation)を参照してください。
```
```suggestion
詳細については、[コンテキスト伝搬](/docs/concepts/context-propagation)を参照してください。
```

ymotongpoo commented on content/ja/docs/languages/_includes/propagation.md:
```diff
@@ -0,0 +1,21 @@
+
+{{ if not $lang }}
+
+コンテキスト伝播を理解するには、コンテキストと伝播という2つの独立した概念を理解する必要があります。
```
```suggestion
コンテキスト伝搬を理解するには、コンテキストと伝搬（プロパゲーション）という2つの独立した概念を理解する必要があります。
```

ymotongpoo commented on content/ja/docs/languages/js/_index.md:
```diff
@@ -0,0 +1,39 @@
+OpenTelemetry JavaScriptには、ブラウザの公式サポートリストはありません。
+主要ブラウザの現在サポートされているバージョンで動作することを目指しています。
+
+OpenTelemetry JavaScriptは、TypeScriptについてDefinitelyTypeのサポートポリシーに従い、2年のサポートウィンドウを設定しています。
```
```suggestion
OpenTelemetry JavaScriptは、TypeScriptについてDefinitelyTypeのサポートポリシーに従い、2年のサポート期間を設定しています。
```

ymotongpoo commented on content/ja/docs/languages/js/context.md:
```diff
@@ -0,0 +1,228 @@
+
+## コンテキストマネージャー {#context-manager}
+
+context APIが動作するためには、コンテキストマネージャーに依存します。
```
```suggestion
コンテキストAPIが動作するためには、コンテキストマネージャーに依存します。
```

ymotongpoo commented on content/ja/docs/languages/js/propagation.md:
```diff
@@ -0,0 +1,523 @@
+サービス間の通信に使用するライブラリに対応する計装ライブラリが存在しない場合があります。
+または、そのようなライブラリが存在していても満たせない要件がある場合があります。
+
+コンテキストを手動で伝搬する必要がある場合は、[context API](/docs/languages/js/context)を使用できます。
```
```suggestion
コンテキストを手動で伝搬する必要がある場合は、[コンテキストAPI](/docs/languages/js/context)を使用できます。
```

ymotongpoo commented on content/ja/docs/languages/js/propagation.md:
```diff
@@ -0,0 +1,523 @@
+
+これは、サーバーとクライアントコードがライブラリである場合に特に重要です。
+ライブラリはOpenTelemetry APIのみを使用するべきだからです。
+その理由を理解するには、[ライブラリに計装を追加する方法のコンセプトページ](/docs/concepts/instrumentation/libraries/)をお読みください。
```
```suggestion
その理由を理解するには、[ライブラリに計装を追加する方法のコンセプトページ](/docs/concepts/instrumentation/libraries/)を確認してください。
```

丁寧にしすぎない

ymotongpoo commented on content/ja/docs/languages/js/propagation.md:
```diff
@@ -0,0 +1,523 @@
+
+## 次のステップ {#next-steps}
+
+伝搬についてさらに学ぶには、[Propagators API仕様](/docs/specs/otel/context/api-propagators/)をお読みください。
```
```suggestion
伝搬についてさらに学ぶには、[Propagators API仕様](/docs/specs/otel/context/api-propagators/)を確認してください。
```

ymotongpoo commented on content/ja/docs/languages/js/serverless.md:
```diff
@@ -0,0 +1,301 @@
+
+以下では、OpenTelemetryでLambdaラッパーを使用してAWS Lambda関数を手動で計装し、設定されたバックエンドにトレースを送信する方法を示します。
+
+プラグアンドプレイのユーザーエクスペリエンスに興味がある場合は、[OpenTelemetry Lambda Layers](https://github.com/open-telemetry/opentelemetry-lambda)を参照してください。
```
```suggestion
プラグアンドプレイのユーザー体験に興味がある場合は、[OpenTelemetry Lambda Layers](https://github.com/open-telemetry/opentelemetry-lambda)を参照してください。
```

ymotongpoo reviewed (changes requested):
1箇所だけ追加コメントしました

chalin commented:
Thanks all!
Thanks @vitorvasc for your help here.
Ideally the benchmark code and styles should be factored out, but this is ok for now.
